src/disco/components/Addon/index.js

```diff
@@ -121,31 +123,58 @@ export class AddonBase extends React.Component {
   }
 
   getThemeImage() {
-    const { getBrowserThemeData, i18n, name, previewURL } = this.props;
-    if (this.props.type === ADDON_TYPE_THEME) {
+    const { type } = this.props;
```

Comment:
I *think* `type` comes from the `addon` prop. I know it is available directly but that's because we spread the whole `addon` if I remember correctly. I think it would be slightly better to retrieve `type` from the `addon`.

This comment also applies to `name`, `previews` and probably `previewURL` too.